.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,64 @@
+name: Build and Test
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types:
+      - opened
+      - synchronize
+jobs:
+  build-and-test:
+    strategy:
+      matrix:
+        ghc: ["8.0.2", "8.2.2", "8.4.4", "8.6.5", "8.8.3", "8.10.1"]
+        os: [ubuntu-latest]
+        include:
+          - os: macos-latest
+            ghc: "8.10.1"
+          - os: ubuntu-16.04
+            ghc: "7.10.3"
+
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Zsh
+        run: |
+          if [ "$RUNNER_OS" == "Linux" ]; then
+            sudo apt-get update
+            sudo apt-get install zsh
+          else
+            brew update
+            brew install zsh
+          fi
+      - uses: actions/setup-haskell@v1.1.4
```

Comment:
@ibarraeit looks like this action is no longer maintained:

![image](https://user-images.githubusercontent.com/2049686/118173873-be88e700-b3f3-11eb-910a-601b8e9a4521.png)

I think you could use https://github.com/haskell/actions instead

Reply:
Great catch! Thanks @sestrella